docs/CHANGELOG_ARCHIVE.md

```diff
@@ -0,0 +1,3731 @@
+- Correct Broken Swift Markdown Docs [\#2600](https://github.com/powerhome/playbook/pull/2600) ([jasoncypret](~https://github.com/jasoncypret))
+- Titleize docs Page Titles [\#2594](https://github.com/powerhome/playbook/pull/2594) ([jasoncypret](https://github.com/jasoncypret))
+- Bump ci-kubed@v6.7.0 [\#2593](https://github.com/powerhome/playbook/pull/2593) ([mfilipe](https://github.com/mfilipe))
+
+[Full Changelog](https://github.com/powerhome/playbook/compare/12.25.0...12.26.0)
+
+# Styling typography just got easier!
+##### June 9, 2023
+
+![12 25 0](https://github.com/powerhome/playbook/assets/73710701/6db5b3e0-f499-446d-965a-0e00b0c31931)
+
+Sometimes you don't need all the styles that ship with the Date kit.
+We have added a simple boolean prop called "unstyled" that puts control in your hands.
+You can now wrap our Date kit with any of our typography kits (Title, Body, Caption, etc) and the date kit will inherit that component's styles! 🎉
+
+[12.25.0](https://github.com/powerhome/playbook/tree/12.25.0) full list of changes:
+
+**Kit Enhancements:**
+
+- Unstyled Variant for Date and Time Kits [\#2592](https://github.com/powerhome/playbook/pull/2592) ([gavhuang](https://github.com/gavhuang))
+- Bold Option for Title 3 [\#2579](https://github.com/powerhome/playbook/pull/2579) ([nickamantia](https://github.com/nickamantia))
+
+**Improvements:**
+
+- Changelog Improvements [\#2591](https://github.com/powerhome/playbook/pull/2591) ([jasoncypret](https://github.com/jasoncypret))
+- Separation of Markdown & PBDocs Dependencies from Playbook Library [\#2527](https://github.com/powerhome/playbook/pull/2527) ([jasoncypret](https://github.com/jasoncypret))
+
+# Responsive Components in Rapid Time: Introducing Global Responsive Spacing Props for React and Rails Components.
+##### May 26, 2023
+
+![titleboldandresponsivespacing](https://github.com/powerhome/playbook/assets/9158723/9741a86c-5439-4081-b197-0101079c8575)
+
+With all components, you can specify any of our spacing props values at certain screen sizes. For example, let's say I wanted the Badge component to have a small padding on phones, and a medium size padding on medium screens and higher. It's as easy as this:
+
+```jsx
+<Badge padding={{ xs: "sm", md: "md"  }} />
+```
+
+Give it a try and let us know what you think!
+
+
+[12.24.0](https://github.com/powerhome/playbook/tree/12.24.0) full list of changes:
+
+**Kit Enhancements:**
+- Title 2 Defaults to Bold [\#2559](https://github.com/powerhome/playbook/pull/2559) ([jasperfurniss](https://github.com/jasperfurniss))
+
+**Improvements:**
+
+- Responsive Spacing global prop [\#2563](https://github.com/powerhome/playbook/pull/2563) ([nickamantia](https://github.com/nickamantia))
+- Creating an example Swift kit doc [\#2550](https://github.com/powerhome/playbook/pull/2550) ([carloslimasd](https://github.com/carloslimasd))
+
+[Full Changelog](https://github.com/powerhome/playbook/compare/12.23.0...12.24.0)
```

Comment:
Lets start the archive here and make sure the other content remains in the changelog